crates/adapter/src/fastly/config_store.rs

```diff
@@ -0,0 +1,25 @@
+use super::fastly_dictionary;
+use super::FastlyStatus;
+
+pub type ConfigStoreHandle = u32;
+
+#[export_name = "fastly_config_store#open"]
+pub fn open(
+    name: *const u8,
+    name_len: usize,
+    store_handle_out: *mut ConfigStoreHandle,
+) -> FastlyStatus {
+    fastly_dictionary::open(name, name_len, store_handle_out)
+}
+
+#[export_name = "fastly_config_store#get"]
+pub fn get(
+    store_handle: ConfigStoreHandle,
+    key: *const u8,
+    key_len: usize,
+    value: *mut u8,
+    value_max_len: usize,
+    nwritten: *mut usize,
+) -> FastlyStatus {
+    fastly_dictionary::get(store_handle, key, key_len, value, value_max_len, nwritten)
+}
```

Comment:
These should probably be implemented in their own right, rather than delegating to the `fastly_dictionary` calls. Maybe that ends up being the same code right now in Viceory, but there's different behavior on the fleet where callers of `fastly_config_store` do not have an implicit fallback to the legacy edge dictionaries.

Reply:
I've added that interface back to `compute.wit`, and implemented it in terms of the dictionary api, which matches the current wiggle implementation in viceory.